<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgDetectableProperties</class>
 <widget class="QDialog" name="CQDlgDetectableProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detectable Properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QCheckBox" name="qqUltrasonic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Ultrasonic</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqInductive">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Inductive</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqCapacitive">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Capacitive</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqInfrared">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Infrared</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="qqLaser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Laser</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="qqOkCancel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>75</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqUltrasonic</tabstop>
  <tabstop>qqInductive</tabstop>
  <tabstop>qqInfrared</tabstop>
  <tabstop>qqCapacitive</tabstop>
  <tabstop>qqLaser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
